src/main/scala/DelayFilter.scala

```diff
@@ -0,0 +1,18 @@
+package EffectBox
+
+import chisel3._
+
+
```

Comment:
Do we need this? This is just an easy instantiation of the fir-filter, so we could  use that instead if so. And I guess 1 cycle delay is not something we typically need. Delay typically needs BRAM and not distributed RAM. I also believe shifting every register in such memory would be unnecessary.

Reply:
Could be cool to use the FIR-filter module to get some easy low pass filter though. (Only input Seq(0.5.U, 0.5.U) I guess?)